.ci/write-dockerfile.sh

```diff
@@ -272,13 +272,14 @@ $ADD src/Pipfile.m4 src/pyproject.toml src/requirements.txt.m4 src/setup.cfg.m4
 $ADD m4 /new/m4
 $ADD pkgs /new/pkgs
 $ADD build /new/build
-$ADD .ci /new/.ci
 $ADD .upstream.d /new/.upstream.d
+ADD .ci /.ci
 RUN if [ -d /sage ]; then                                               \
```

Comment:
What's the difference between `$ADD` and `ADD`?

Reply:
Ah it is `ADD="ADD $__CHOWN"`...

Reply:
Why here `ADD` and `$ADD` below?

Reply:
Thanks for catching this; fixed in f4f0a0ca80f